.kokoro/java11/continuous.cfg

```diff
@@ -19,3 +19,9 @@ env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
     value: "github/java-docs-samples/.kokoro/tests/run_tests.sh"
 }
+
+# Only diff from previous commit
```

Comment:
So your changing the meaning?

Reply:
This is changing the scope of the continuous tests from "test everything every commit" to "test everything changed in the last commit". In python-docs-samples, they switched to this model to prevent every RenovateBot PR from triggering a full test suite. There are a lot less conflicts if there isn't a 3-4 hour suite run everytime a commit is made.